koji/collection-mash-split.py

```diff
@@ -400,12 +400,15 @@ def main():
 
         CONFIG["comps_path"] = "/mnt/koji/mash/comps/foreman"
 
-        mashes = [MashConfig(collection, version, "rhel7-dist", "rhel7", "RHEL/7")]
         if collection == 'foreman' and version not in ('3.1', '3.0', '2.5'):
             modulemd_suffix = 'el8'
         else:
             modulemd_suffix = None
-        mashes.append(MashConfig(collection, version, "el8", "el8", "RHEL/8", modulemd_suffix=modulemd_suffix, modulemd_version=modulemd_version, modulemd_context=modulemd_context))
+
+        mashes = MashConfig(collection, version, "el8", "el8", "RHEL/8", modulemd_suffix=modulemd_suffix, modulemd_version=modulemd_version, modulemd_context=modulemd_context)
+
+        if LooseVersion(version) <= LooseVersion("3.3"):
+            mashes.append([MashConfig(collection, version, "rhel7-dist", "rhel7", "RHEL/7")])
```

Comment:
And this shouldn't be a list:
```suggestion
            mashes.append(MashConfig(collection, version, "rhel7-dist", "rhel7", "RHEL/7"))
```